src/test/java/com/codeborne/selenide/UnitTest.java

```diff
@@ -0,0 +1,10 @@
+package com.codeborne.selenide;
+
+import org.assertj.core.api.WithAssertions;
+
+/**
+ * @author Aliaksandr Rasolka
+ * @since 4.12.2
+ */
+public class UnitTest implements WithAssertions {
```

Comment:
I don't understand what the metaphor should stand behind this class.
In any case, it should not appear in the main package of selenide, because as far as I know some of Selenide users - there will be many of them trying to use this class in every crazy and terrible way they can find :)